project/project.go

```diff
@@ -63,14 +63,24 @@ func (p *Project) Parse() error {
 
 	p.Name = p.context.ProjectName
 
-	if p.context.ComposeFile == "-" {
-		p.File = "."
+	if len(p.context.ComposeFiles) == 1 && p.context.ComposeFiles[0] == "-" {
+		p.Files = []string{"."}
 	} else {
-		p.File = p.context.ComposeFile
+		p.Files = p.context.ComposeFiles
 	}
 
 	if p.context.ComposeBytes != nil {
-		return p.Load(p.context.ComposeBytes)
+		for i, composeBytes := range p.context.ComposeBytes {
+			if i < len(p.context.ComposeFiles) {
```

Comment:
What about :

```go

			file := ""
			if i < len(p.context.ComposeFiles) {
				file = p.Files[i]
			}
			if err := p.load(file, composeBytes); err != nil {
				return err
			}
```

But then, I would refactor the `Load` method to take a file *string* and not create `load` (if not used elsewhere).